### ISSUE TITLE
Show Comment Type

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/meme-2023#43]
Joshua writes:
> For clarity on the comments, please add the "type" in some manner above the displayed comment. In Evan's mockup it reads "Critera: Clarity" but I think we want to just assume it is "Clarity" or "Evidence" using the existing label field. If we really want criteria in there we'd change the label "Criteria: Clarity" so that the same change / heading works in both MEME and net.create and for non-criteria things. No rush since I'll demo as-is.

_Originally posted by @jdanish in https://github.com/theRAPTLab/meme-2023/issues/41#issuecomment-2377299059_

**Comment by benloh** [comment]
"Criteria" was a function of the old comment model.  This is no longer used.

The new comment model has the following format:
```
  {
    slug: 'tellmemore',
    label: 'Tell me more', // comment type label
    prompts: [
      {
        format: 'text',
        prompt: 'Please tell me more', // prompt label
        help: 'Can you tell me more about ... ',
        feedback: ''
      }
    ]
  },
```

Here's how they map:
* `label` is used to display the selection menu, in this case "Tell me more"
![screenshot_1722](https://github.com/user-attachments/assets/eed93eba-db30-4b94-943b-261abb55c02a)

* `prompt` and `help` is used to display the question and help text in edit mode, in this case "Please tell me more"
![screenshot_1723](https://github.com/user-attachments/assets/fa9143c7-cfa9-4d5f-8706-aaf41549d3b1)

* In view mode, currently only the `prompt` is displayed:
![screenshot_1724](https://github.com/user-attachments/assets/db0c9710-48b1-4fef-9302-04d3c69c2a15)

Note that for a comment type that has multiple prompts, they are also shown, e.g. for
```
  {
    slug: 'source',
    label: 'Source', // comment type label
    prompts: [
      {
        format: 'text',
        prompt: 'Is this well sourced?', // prompt label
        help: 'Yes/No',
        feedback: ''
      },
      {
        format: 'text',
        prompt: 'Changes', // prompt label
        help: 'What about the sourcing could be improved?',
        feedback: ''
      }
    ]
  }
```
* `label`
![screenshot_1725](https://github.com/user-attachments/assets/3b355016-7a97-428e-b75f-135224f351c9)

* `prompt` 1  and  `prompt` 2
![screenshot_1726](https://github.com/user-attachments/assets/92008525-7c77-4e4d-a239-ae057805828d)



So this kind of depends on what you want.  We can show "Type:" if you want to show the selected type, e.g.
```
Type: Tell me more             -- mapping to `label`
Please tell me more            -- mapping to `prompt`
[ This is my comment ]
```

and for the "Source" comment type
```
Type: Source                       -- mapping to `label`
Is this well sourced?               -- mapping to `prompt` 1
[ This is my comment ]
Changes                              -- mapping to `prompt` 2
[ The changes are... ]
```

I think that's probably closest to how things used to work.  So if you wanted to restore the old criteria, you would define this:
```
  {
    slug: 'criteria',
    label: 'Criteria', // comment type label
    prompts: [
      {
        format: 'text',
        prompt: 'Clarity', // prompt label
        help: 'How clear is the text?',
        feedback: ''
      },
      {
        format: 'text',
        prompt: 'Validity', // prompt label
        help: 'How valid is the text?',
        feedback: ''
      }
    ]
  }
```
and then the display would look like this:
```
Type: Criteria                         -- mapping to `label`
Clarity                                  -- mapping to `prompt` #1
[ This is my comment ]
Validity                                -- mapping to `prompt` #2
[ The changes are... ]
```


@jdanish Is this what you want?

**Comment by jdanish** [comment]
> Type: Criteria                         -- mapping to `label`
Clarity                                  -- mapping to `prompt` #1
[ This is my comment ]
Validity                                -- mapping to `prompt` #2
[ The changes are... ]

Yes, something like this because right now the prompts are all at the same level so the "grouping" is unclear. If we have the Label called out, and then the prompts are intended slightly because of the dots it'll be clear the hierarchy.

**Comment by jdanish** [comment]
I don't think this changes anything, but because we can't do branching, we'll likely have tables for Evidence, Clarity, etc., and then they'll have slightly different formats but this works.

**Comment by benloh** [comment]
@jdanish OK, just making sure this is what you had in mind...is this right?

![screenshot_1727](https://github.com/user-attachments/assets/3104b0a2-9ca9-429d-b4cc-6234fd477e41)

**Comment by jdanish** [comment]
Yes thanks!

**Comment by benloh** [comment]
Implemented with 6b4df6fdaa0f71108030fbf48c413cf526413633
